setup.py

```diff
@@ -43,6 +43,7 @@
     "pkgmt",
     "twine",
     # tests
+    # DuckDB 0.10.1 breaking Sqlalchemy v1 tests: https://github.com/ploomber/jupysql/issues/1001 # noqa
```

Comment:
please rebase, this comment has been removed in the latest code

Reply:
Indeed